custom_components/google_home/sensor.py

```diff
@@ -60,6 +61,17 @@ async def async_setup_entry(
     return True
 
 
+class DeviceAttributes(TypedDict):
+    """Typed dict for device attributes"""
+
+    device_name: str
+    auth_token: Optional[str]
+    ip_address: Optional[str]
+    hardware: Optional[str]
+    available: bool
+    integration: str
```

Comment:
I suggest we move out `TypedDict` to a separate file like `types.py`. It would give it a bit better structure